src/module/base.js

```diff
@@ -56,12 +57,37 @@ const modifyData = (container) => {
   });
 };
 
+const checkBox = (container) => {
+  container.addEventListener('click', (element) => {
+    if (element.target.classList.contains('list-check')) {
+      const checkBoxAction = element.target;
+      const dataLi = checkBoxAction.parentElement.parentElement.parentElement;
+      const data = dataLi.querySelector('.data-text');
+      data.classList.toggle('text-dec');
+      Modify.doneCheck(dataLi);
```

Comment:
- [optional] This review is recommended, as it is your project that performs all the basic requirements. Still, personally, i noticed, that the behavior of the completed task changed, and won't strike through even if it is completed. This error can be replicated by
     -  adding three tasks. 
     - mark two of them as completed, 
     - then add another task making it four, 
     - now try to change the completed state of your task, and you will notice the little glitch.

**View the image showing it below.**
- Frank is checked but does not strike through. If you need more clarity you can reach me on slack as `kanu mike chibundu`

![image](https://user-images.githubusercontent.com/58771507/201171450-7bd59f55-1454-4505-a568-027d72b44fc9.png)